utils/mocks.go

```diff
@@ -0,0 +1,51 @@
+package utils
```

Comment:
move this file to `tests/utils` because it contains `t *testing.T` please rename this file to `test_mocks.go`

Reply:
this caused circle import.
the unit tests are using this mock, so this struct can be in theire directory.